docs/guides/references/migration-guide.mdx

```diff
@@ -8,15 +8,79 @@ This guide details the changes and how to change your code to migrate to Cypress
 version 13.0.
 [See the full changelog for version 12.0](/guides/references/changelog#12-0-0).
 
+### `.should()` and `.and()` are now query commands
+
+In Cypress 13, [assertions](/api/commands/should) are now queries. Most tests
+should continue to operate as before, but there are some differences to the
+previous behavior to be aware of.
+
+#### Why Cypress made this change
+
+Assertions being queries allows for extremely flexible and fluent chaining of
+Cypress commands. See
+[#25134](https://github.com/cypress-io/cypress/issues/25134) for an example.
+This change also lays the groundwork for (freely chaining Cypress
+commands)[https://github.com/cypress-io/cypress/issues/25655] and eventually
+native [`async / await`](https://github.com/cypress-io/cypress/issues/1417)
+support.
+
+#### Assertions now retry after _all_ commands.
```

Comment:
This isn't true. There are several Cypress commands that only execute once.

Reply:
okay - mis-read this, however,
> However, assertions now execute
independently of the previous command, meaning they always retry.

This needs more context to how it behaved previously and how this is influenced by query commands.

Reply:
Added more text explaining how it used to behave and how it behaves now, including an additional example.